lib/style-transform.js

```diff
@@ -1,156 +1,85 @@
 const stylis = require('stylis')
 
-/**
- * @type {RegExp}
- *
- * matches :global($1) capturing $1
- */
-const regExpReplaceGlobal = /:global\((.*)\)/g
+function disableNestingPlugin(...args) {
+  let [context, , , parent = [], line, column] = args
```

Comment:
for the record the reason is that xo (the linter) complains about too many args and unused ones